lockfile_generator/src/dotnet.rs

```diff
@@ -0,0 +1,24 @@
+//! JavaScript yarn ecosystem.
+
+use std::path::Path;
+use std::process::Command;
+
+use crate::Generator;
+
+pub struct Dotnet;
+
+impl Generator for Dotnet {
+    fn lockfile_name(&self) -> &'static str {
+        "packages.lock.json"
```

Comment:
Not as far as I know, since we always generate the file for one specific manifest so it should output to `packages.lock.json` and leave the other lockfiles unaffected.

That said, I haven't actually been able to generate a `packages.*.lock.json` and I'm not entirely sure what it takes to generate one.